```
/../src/suan-2.5.3-6d22bdc3fe.clean/configure: line 11557: /mnt/vcpkg-ci/installed/x64-linux/tools/libgpg-error/bin/gpg-error-config: No such file or directory
./../src/suan-2.5.3-6d22bdc3fe.clean/configure: line 11568: test: : integer expression expected
./../src/suan-2.5.3-6d22bdc3fe.clean/configure: line 11571: test: : integer expression expected
./../src/suan-2.5.3-6d22bdc3fe.clean/configure: line 11578: test: : integer expression expected
configure: error: libgpg-error was not found
```

--- a/ports/libgpg-error/portfile.cmake
+++ b/ports/libgpg-error/portfile.cmake
@@ -40,7 +40,7 @@ if(VCPKG_TARGET_IS_WINDOWS)
 
     vcpkg_install_msbuild(
         USE_VCPKG_INTEGRATION
-        SOURCE_PATH ${SOURCE_PATH}
+        SOURCE_PATH "${SOURCE_PATH}"
         PROJECT_SUBPATH SMP/libgpg-error.sln
         PLATFORM ${TRIPLET_SYSTEM_ARCH}
         LICENSE_SUBPATH COPYING.LIB
@@ -81,7 +81,7 @@ else()
 
     vcpkg_configure_make(
         AUTOCONFIG
-        SOURCE_PATH ${SOURCE_PATH}
+        SOURCE_PATH "${SOURCE_PATH}"
         OPTIONS
             --disable-tests
             --disable-doc
@@ -91,7 +91,8 @@ else()
     vcpkg_install_make()
     vcpkg_fixup_pkgconfig() 
     vcpkg_copy_pdbs()
-
+    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/libgpg-error/bin/gpg-error-config" "${CURRENT_INSTALLED_DIR}" "`dirname $0`/../../..")
+    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/libgpg-error/debug/bin/gpg-error-config" "${CURRENT_INSTALLED_DIR}" "`dirname $0`/../../../..")
     file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/share/${PORT}/locale" "${CURRENT_PACKAGES_DIR}/debug/share")
     file(INSTALL "${SOURCE_PATH}/COPYING.LIB" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
 endif()

--- a/ports/libgpg-error/vcpkg.json
+++ b/ports/libgpg-error/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libgpg-error",
   "version": "1.42",
-  "port-version": 1,
+  "port-version": 2,
   "description": "A common dependency of all GnuPG components",
   "homepage": "https://gnupg.org/software/libgpg-error/index.html",
   "supports": "!(windows & (arm | arm64))"

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3462,7 +3462,7 @@
     },
     "libgpg-error": {
       "baseline": "1.42",
-      "port-version": 1
+      "port-version": 2
     },
     "libgpod": {
       "baseline": "2019-08-29",

--- a/versions/l-/libgpg-error.json
+++ b/versions/l-/libgpg-error.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "f6c78e927a70ff136576abd61f1125861824bfd2",
+      "version": "1.42",
+      "port-version": 2
+    },
+    {
       "git-tree": "b5abbf9cf832337e202ff5d8d4296025b47be1a0",
       "version": "1.42",
       "port-version": 1

